updated-file-/adev/src/content/guide/routing/lifecycle-and-events.md

portneon opened the pull request:
docs: update navigation event example to use event.code

Update the code sample to use `event.code` instead of `event.reason` as
requested in issue #65232. This aligns the example with the updated API and
prevents confusion for developers using the NavigationCancellationCode enum.

The existing use of `event.reason` in the console.warn example is preserved
because it is the correct and intentional usage for that specific case.

Fixes #65232

SkyZeroZx commented on adev/src/content/guide/routing/lifecycle-and-events.md:
```diff
@@ -175,8 +176,8 @@ export class ErrorHandlerComponent {
       } else if (event instanceof NavigationCancel) {
-        console.warn('Navigation cancelled:', event.reason);
-        if (event.reason === NavigationCancellationCode.GuardRejected) {
+        console.warn('Navigation cancelled:', event.code);
```
I think this should be preserved for the example  , specifically on line 179 where `event.reason` is using it correctly for console.warn.
Otherwise, it looks good to me.

SkyZeroZx replied:
It appears this change that was made has been reversed; you can check it @portneon.

portneon commented:
I have made the suggest change also I have signed the Google CLA #65232 this is the final changes.Please have a look on it and sorry for wasting your time on all the mistakes that i have made.

SkyZeroZx commented:
The CLA verification is still failing; please ensure it's the same email address as your GitHub user account.
Also, the commit needs to be a single. You can try rebasing and creating a single [commit with the correct format](https://github.com/angular/angular/blob/main/contributing-docs/commit-message-guidelines.md) .

portneon committed:
docs(router): update navigation event example to use event.code

Update the code sample to use `event.code` instead of `event.reason` for cancellation checks, while preserving `event.reason` in the console.warn line. This aligns the example with the NavigationCancellationCode enum and prevents confusion.

Fixes #65232
